src/js/CheckBox/index.ts

```diff
@@ -96,10 +95,11 @@ class CheckBox extends Control {
         if(!this._props.items) {
             this._props.items = []
         }
-        this._props.items.push(item)
-        if (this._validator()) {
-            throw new Error(this._validator())
+        const itemsToCheck = this._props.items.concat(item);
```

Comment:
Not sure why using `concat` here. As I understand item is and `ItemData` so this should be:
```javascript
const itemsToCheck = this._props.items.push(item);
```

Reply:
I use concat() becasue push() method change original "this._props.items"  value so when error happen in this._validator(), we should restore "this._props.items".

Reply:
@phongnhat19 
I use concat() because push() method change original "this._props.items"  value so when error happen in this._validator(), we should restore "this._props.items".

Reply:
@youtr-s 
If that's the purpose, I think we should clone `this._props.items` into a new array and operate on that.

Although `concat` here will still work with an `ItemData` parameter, it may lead to some confusing for other developers as it is stated [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat): 

> The `concat()` method is used to merge two or more arrays.